File: MCRRTImp/src/test/java/infrastructure/PolarBasedImp/CircleObstacle.java
package infrastructure.PolarBasedImp;

import lab.mars.RRTBase.Obstacle;

public class CircleObstacle implements Obstacle<Polar> {

    Polar centroid;

    double radius;

    public CircleObstacle(Polar centroid, double radius) {
        this.centroid = centroid;
        this.radius = radius;
    }

    @Override
    public boolean contains(Polar o) {
        return false;
    }
}

File: MCRRTImp/src/test/kotlin/infrastructure/PolarBasedImp/CircleObstacle.kt
package infrastructure.PolarBasedImp

import lab.mars.MCRRTImp.base.Obstacle


class CircleObstacle(internal var centroid: Polar, internal var radius: Double) : Obstacle<Polar> {

    override operator fun contains(o: Polar): Boolean {
        return false
    }
}
